contracts/axelar-gateway/src/event.rs

```diff
@@ -30,9 +30,9 @@ pub(crate) fn execute_contract_call(env: &Env, message_id: String) {
     env.events().publish(topics, ());
 }
 
-pub(crate) fn rotate_signers(env: &Env, signers: WeightedSigners) {
+pub(crate) fn rotate_signers(env: &Env, signers_hash: BytesN<32>, signers: WeightedSigners) {
```

Comment:
change the event in a separate PR for clarity since this is an external dependency in Amplifier and relayer